src/SearchResults.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+const headings = [
+  "id",
+  "title",
+  "firstName",
+  "surname",
+  "email",
+  "roomId",
+  "checkInDate",
+  "checkOutDate",
+  "numberOfNights"
+];
+
+function SearchResults(props) {
+  return (
+    <table>
+      <thead>
+        <tr>
+          {headings.map(heading => {
+            return <th scope="col">{heading}</th>;
+          })}
+        </tr>
+      </thead>
+      <tbody>
+        {props.bookings.map(booking => {
+          return (
+            <tr>
+              <th scope="row">{booking.id}</th>
```

Comment:
`<th>` is table cell for header row. `<td>` tag is used for all non header row table cells